tincr/io/library/genCellLibrary.tcl

```diff
@@ -833,13 +939,15 @@ proc ::tincr::create_xml_cell_library { {part xc7a100t-csg324-3} {filename ""} {
     
     set xml_out [open $filename w]
 
+    # print the RapidSmith 2.0 license to the file
+    print_rapidSmith_license $xml_out
```

Comment:
The files that are generated are usually included in the RapidSmith repository. I agree that it is a little strange that we are including a RS2 license in Tincr, but I don't want to copy and paste the license whenever I update the cell library. @nelsobe what do you think?